src/components/figures/chart/chartJs/chartjs.ts

```diff
@@ -60,10 +64,7 @@ export class ChartJsComponent extends Component<Props, SpreadsheetChildEnv> {
     } else {
       this.chart!.data.datasets = [];
     }
-    this.chart!.config.options!.plugins!.tooltip = chartData.options!.plugins!.tooltip;
-    this.chart!.config.options!.plugins!.legend = chartData.options!.plugins!.legend;
-    this.chart!.config.options!.scales = chartData.options?.scales;
-    // ?
+    this.chart!.config.options = chartData.options;
```

Comment:
Can't we have something we would like to keep (not overwrite) in the config.options ? Shouldn't we do something like
```js
this.chart!.config.options = {...this.chart!.config.options, ...chartData.options}
```
to be sure ?

Reply:
Is that really something we want ? I feel like we want the options in the actual ChartJS chart to exactly match the options in the chart Runtime, and not keep leftover options from previous runtimes

Reply:
maybe @LucasLefevre will have an option on this but I think it was more about changing the least options possible?

Reply:
I don't know why we only changed specifically tooltip, legend and scales rather then reset all the options.
Overriding the entire options seem good.
As Adrien said, the options from the runtime should be correct and complete, we don't want/need to preserve the ones chart.js has